static/midcom.grid/jqGrid.custom.js

```diff
@@ -781,6 +781,14 @@ const midcom_grid_row_actions = {
     process: function(button, action, config) {
         button.attr('disabled', 'disabled');
         var id = button.parent().parent().attr('id');
+
+        if (action === 'send_by_mail') {
```

Comment:
hmm, it's a bit unfortunate to have invoice-specific logic in the generic grid component. Isn't there a way to have this logic in the o.o.invoices component?